db/migrate/20231107130000_add_open_data_role_to_upvs_schema_by_default.rb

```diff
@@ -0,0 +1,7 @@
+class AddOpenDataRoleToUpvsSchemaByDefault < ActiveRecord::Migration[6.0]
+  def change
+    execute 'GRANT USAGE ON SCHEMA upvs to open_data'
+    execute 'GRANT SELECT ON ALL TABLES IN SCHEMA upvs TO open_data'
+    execute 'ALTER DEFAULT PRIVILEGES IN SCHEMA upvs GRANT SELECT ON TABLES TO open_data'
```

Comment:
Neviem ci to chceme uplne vsetko default otvarat, nerobili sme to teraz vzdy per-table?

Reply:
No, itms, metais a eid22 (mimo harvester) majú takto default. Čiže všetko, čo ťahá harvester, ktorý je public repo.